bench/wtperf/wtperf.c

```diff
@@ -1397,19 +1583,21 @@ monitor(void *arg)
 
 		(void)fprintf(fp,
```

Comment:
When you add a field to this `fprintf`, you'll need to followup with a change to a couple of Jenkins jobs that "know" what the fields mean.  See https://jira.mongodb.org/browse/WT-4967 for the last time this happened.  It's probably best to open a separate ticket to fix the Jenkins part, and just plan to do that as soon as this ticket merges.

Reply:
I will create another Jira ticket similar to WT-4957 after it is merged.

Reply:
Created a new Jira ticket to correct Jenkins fail jobs 
https://jira.mongodb.org/browse/WT-5062